docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -8,13 +8,18 @@ Elastic Security allows any event detected by Elastic Endpoint to be analyzed us
 [[find-events-analyze]]
 === Find events to analyze
 
-You can only visualize events triggered by hosts configured with the Elastic Endpoint Security Integration. In KQL, this translates to any event with the `agent.type` set to `endpoint`.
+You can only visualize events triggered by hosts configured with the Elastic Endpoint Security Integration or any sysmon data for `winlogbeat`. 
+
+In KQL, this translates to any event with the `agent.type` set to either: 
+
+- `endpoint`
+- `winlogbeat` with `event.module` set to `sysmon`.
 
 To access events that can be visually analyzed:
 
 1. In Elastic Security, select **Hosts** > **Events**. A list of all your host's events appears in the bottom panel of the host's view.
 
-2. Create a KQL query that filters all `endpoint` detected events by entering `agent.type: endpoint and process.entity_id : *` into the KQL search bar, and then select **Update**.
+2. Create a KQL query that filters all `endpoint` detected events by entering `agent.type: endpoint and process.entity_id : *` or `agent.type: winlogbeat, event.modile: sysmon, process.entity_id : *` into the KQL search bar, and then select **Update**.
```

Comment:
**or `agent.type: winlogbeat and event.module: sysmon and process.entity_id : *`**